### PR TITLE
[CI:BUILD] packit: tag @containers/packit-build team on copr build failure

### PR DESCRIPTION
**Pull request by lsm5**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by lsm5** [comment]
@edsantiago @cevich @Luap99 @mheon @rhatdan PTAL

**Comment by lsm5** [comment]
@haircommander @saschagrunert ping, might be worth doing on conmon-rs too. PTAL.

File: .packit.yaml [hunk 1]
```diff
@@ -16,8 +16,10 @@ actions:
 jobs:
   - job: copr_build
     trigger: pull_request
+    notifications:
+      failure_comment:
+        message: "Ephemeral COPR build failed. @lsm5 please check."
```
Comment (TomSweeneyRedHat) [inline .packit.yaml:21]: Can you create a GitHub Team or Organization and use that instead?  Even if that group only has you in it.

Comment (lsm5) [reply .packit.yaml:21]: hmm, don't know if the packit feature works for teams but we can try. We'll also need something similar for the revdep failure tag @martinpitt

Comment (martinpitt) [reply .packit.yaml]: It's not a feature of packit. You can specify whichever text you want in the notification. The `@user` is a GitHub feature. But I'm not sure if e.g. tagging the @cockpit-project org actually works (that would be way too big, just for testing -- if it works, sorry everyone for the noise!)

Comment (lsm5) [reply .packit.yaml]: done

Comment (lsm5) [reply .packit.yaml]: @martinpitt maybe a cockpit team with the three of you would work? I created @containers/packit-build just now.

Comment (lsm5) [reply .packit.yaml]: @TomSweeneyRedHat cockpit org/team can be handled in a separate PR.

**Comment by TomSweeneyRedHat** [comment]
LGTM if that team thing works.

File: .packit.yaml (hunk 1 above)
Comment (lsm5) [reply .packit.yaml]: @martinpitt i could even create a `containers/cockpit-tests` team if that works for you.

Comment (martinpitt) [reply .packit.yaml]: Yup, I'm happy to handle the cockpit notifications separately. But I'm still not convinced that tagging a team actually works -- did anyone from the cockpit-project org actually get this notification? Please react with :+1: here if you did. Thanks

**Review by Luap99 — APPROVED** [review]
/lgtm

File: .packit.yaml (hunk 1 above)
Comment (lsm5) [reply .packit.yaml]: @martinpitt I see both team and org mentions in your latest comment. To be clear, I haven't created any cockpit team in the containers org yet.

Comment (martinpitt) [reply .packit.yaml]: @lsm5 I just asked my colleagues in @cockpit-project, and they did *not* get the notification (that also makes sense -- spam protection). So an org won't work. A team might, let me test that in https://github.com/cockpit-project/bots/pull/5288#issuecomment-1731373705

Comment (martinpitt) [reply .packit.yaml]: Team tagging indeed seems to work :tada:

**Pull request merged**